assets/jsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
-    "baseUrl": "./",
+    "jsx": "react", // Allow command-click open components in a new file in VSCode
```

Comment:
Thank you for this! I tried with your branch on a UIFormField and it didn't work as expected. Is this only for some files?

Reply:
@divyareddynu Try restarting VSCode?   I'll double check too.